src/widget/city_without_overlay.c

```diff
@@ -731,6 +740,73 @@ static int get_highlighted_formation_id(const map_tile *tile)
     return highlighted_formation_id;
 }
 
+#ifdef DRAW_ROUTING
+static void draw_routing(int x, int y, int grid_offset)
+{
+    int tx = map_grid_offset_to_x(grid_offset);
+    int ty = map_grid_offset_to_y(grid_offset);
+    map_routing_distance_grid *distance = map_routing_get_distance_grid();
+    int16_t dist = distance->determined.items[grid_offset];
+    if (!dist) {
+        return;
+    }
+    if (tx == distance->dst_x && ty == distance->dst_y) {
+        int dst_image_id = assets_get_image_id("UI", "Happy God Icon");
+        image_draw(dst_image_id, x + 29 - 10, y + 15 - 10, 0, 1);
+    }
+    char *text[20];
+    sprintf(text, "%d", dist);
+    text_draw_centered(text, x, y, 58, FONT_NORMAL_BLACK, COLOR_WHITE);
+}
+#endif
+
+#ifdef DRAW_HIGHWAY_TERRAIN
```

Comment:
I think this specific debug data is too situational to be useful somewhere else.

I say remove it entirely.

Reply:
Moved to debug.c per discord conversation.